src/api/routes/v1/get-network.ts

```diff
@@ -132,11 +138,19 @@ async function fetchCrawls(host: string): Promise<Crawl> {
  * @param unl - The UNL of the node.
  * @returns Whether the UNL of the node has been seen before.
  */
-async function getNetworkFromUNL(unl: string): Promise<string | undefined> {
+async function getNetworkInfoFromUNL(
```

Comment:
This API endpoint has not been documented in the `ARCHITECTURE.md` file. How am I supposed to use it?

This [URL](https://data.xrpl.org/v1/network/get_network/s2.ripple.com) does not work

<img width="269" alt="image" src="https://github.com/user-attachments/assets/086deee2-fafa-49a3-91cb-5e90ccdc4b5d" />